core/src/main/java/nu/yona/server/messaging/entities/MessageDestination.java

```diff
@@ -5,21 +5,16 @@
 package nu.yona.server.messaging.entities;
```

Comment:
I'm not happy with the changes in this file. This breaks down the domain model for message destinations and reduces the value of the ``MessageDestination`` entity to nearly nothing.